lib/block-supports/background.php

```diff
@@ -85,7 +103,9 @@ function gutenberg_render_background_support( $block_content, $block ) {
 
 			$updated_style .= $styles['css'];
```

Comment:
This is not ideal, and I'm not convinced we should be skipping serialization for for background images as we're doing here:

https://github.com/WordPress/gutenberg/blob/13e5b6d5bfedde39880833200fe922417e36d653/packages/block-editor/src/hooks/style.js#L136-L136

Color gradients output values using the `background` shorthand CSS property. Since serialization is not skipped in this case it will exist in the CSS output along with any `background-image` styles, since the updated CSS is concatenated. 

For the record, here's are some stated reasons why they're being skipped:

https://github.com/WordPress/gutenberg/pull/53934#issuecomment-1756213964

https://github.com/WordPress/gutenberg/pull/53934#issuecomment-1710478769

https://github.com/WordPress/gutenberg/pull/53934#issuecomment-1695271930  (argument against)

Reply:
As it is, where there is an image:

- where a `background: ....;` inline CSS rule exists, it needs to be parsed out of the string to make way for the combined `background-image` value.
- where a preset classname exists, e.g., `has-very-light-gray-to-cyan-bluish-gray-gradient-background`, it needs to be removed in favour of an inline CSS rule that uses the CSS var, e.g., `background-image: var(--wp--preset--gradient--very-light-gray-to-cyan-bluish-gray), url(http://localhost:8888/wp-content/uploads/2024/04/white-bellied-swallow.jpg);`